docs/visualization/lupton_rgb.rst

```diff
@@ -0,0 +1,88 @@
+.. _astropy-visualization-rgb:
+
+*************************
+Creating color RGB images
+*************************
+
+`Lupton et al. (2004)`_ describe an "optimal" algorithm for producing red-green-
+blue composite images from three separate high-dynamic range arrays. This method
+is implemented in `~astropy.visualization.make_lupton_rgb` as a convenience
+wraper function and an associated set of classes to provide alternate scalings.
+The SDSS SkyServer color images were made using a variation on this technique.
+To generate a color PNG file with the default (arcsinh) scaling:
+
+.. _Lupton et al. (2004): http://adsabs.harvard.edu/abs/2004PASP..116..133L
+
+.. plot::
+    :include-source:
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    from astropy.visualization import make_lupton_rgb
+    image_r = np.random.random((100,100))
+    image_g = np.random.random((100,100))
+    image_b = np.random.random((100,100))
+    image = make_lupton_rgb(image_r, image_g, image_b, stretch=0.5)
+    plt.imshow(image)
+
+This method requires that the three images be aligned and have the same pixel
+scale and size. Changing ``minimum`` will change the black level, while
+``stretch`` and ``Q`` will change how the values between black and white are
+scaled.
+
+For a more in-depth example, download the `g`_, `r`_, `i`_ SDSS frames of the
+area around the Hickson 88 group and try the example below (requires the
+`reproject package`_, installable via pip), and compare it with Figure 1 of
```

Comment:
minor comment, and I'm not sure it's important for the docs, but reproject (as most of the affiliated packages) is also available on conda via the astropy channel.